static/src/stylesheets/module/commercial/_hostedbadge.scss

```diff
@@ -5,25 +5,25 @@
     width: 80px;
     z-index: $zindex-content;
 
-    &.hostedbadge--shouldscale {
-        @include mq(desktop) {
-            top: -28px;
-            left: calc(50% - 490px + #{$gs-gutter});
-            width: 132px;
-        }
+    &:hover,
+    &:focus {
```

Comment:
Also, make sure the `:active` style doesn't look weird. I had the problem with `.advert`.